integration/mediation-tests/tests-platform/tests-rabbitmq/src/test/java/org/wso2/carbon/esb/rabbitmq/store/RabbitMQStoreTestCase.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.esb.rabbitmq.store;
+
+import org.wso2.carbon.integration.common.admin.client.LogViewerClient;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.esb.integration.common.utils.Utils;
+import org.wso2.esb.integration.common.utils.clients.axis2client.AxisServiceClient;
+
+import static org.testng.Assert.assertTrue;
+
+/**
+ * This test case tests whether messages can be stored and forwarded in a RabbitMQ message store.
+ */
+public class RabbitMQStoreTestCase extends ESBIntegrationTest {
+    private static LogViewerClient logViewer;
+    private AxisServiceClient client = new AxisServiceClient();
+
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+        super.init();
+        loadESBConfigurationFromClasspath("/artifacts/ESB/store/rabbitmqStoreTest.xml");
+        logViewer = new LogViewerClient(context.getContextUrls().getBackEndUrl(), sessionCookie);
+    }
+
+    @Test(groups = "wso2.esb",
+          description = "Testing guaranteed delivery scenario")
+    public void testRMQStore() throws Exception {
```

Comment:
This doesn't simply test message store, rather the combined action of message store and processor. Shall we rename the test case and the test method accordingly?

Reply:
done